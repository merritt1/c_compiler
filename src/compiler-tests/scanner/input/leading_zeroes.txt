123
09
007
