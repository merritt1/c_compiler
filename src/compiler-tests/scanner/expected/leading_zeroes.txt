loc = 0001:0001-0001:0003     text = 123                      token = [NUMBER              ]     type =   signed  int     value = [123       ]
loc = 0002:0001-0002:0001     text = 0                        token = [NUMBER              ]     type =   signed  int     value = [0         ]
loc = 0002:0002-0002:0002     text = 9                        token = [NUMBER              ]     type =   signed  int     value = [9         ]
loc = 0003:0001-0003:0001     text = 0                        token = [NUMBER              ]     type =   signed  int     value = [0         ]
loc = 0003:0002-0003:0002     text = 0                        token = [NUMBER              ]     type =   signed  int     value = [0         ]
loc = 0003:0003-0003:0003     text = 7                        token = [NUMBER              ]     type =   signed  int     value = [7         ]
